packages/shared/src/styles/themes.scss

```diff
@@ -15,7 +15,7 @@
     --text-lh-l: 1.75;
     --text-lh-m: 1.5;
     --text-lh-s: 1.25;
-    --text-lh-x: 1.4;
+    --text-lh-xs: 1.4;
```

Comment:
isn't `xs` supposed to be smaller than `s`? 😅

Reply:
yes  😅 so I have to do a big refactoring  😅

Reply:
let's do that in this PR then? Better to fix it in one shot rather than accumulate code debt. 😬

Reply:
I did it on this PR. there were only few cases.